### PR TITLE
Show exception messages on media error

### PR DESCRIPTION
**Pull request by maliouris**
Pull Request for Issue #25379

### Summary of Changes

In com_media change _handleError js function in order to show error messages if there are any thrown by an exception, otherwise show some defaults based on error code.

### Testing Instructions

Run:
` npm run dev:com_media`
` npm run build:js`

reload browser with: ` CTL+F5`

Set max file size to 1 and upload a larger file

### Actual result BEFORE applying this Pull Request

Error message will show "You are not authorized"

### Expected result AFTER applying this Pull Request

Error message `This file is too large to upload.`

### Documentation Changes Required

Some other constants may have to be declared to /administrator/language/en-GB as /language/en-GB files are not loaded in the administrator view

### CODE REVIEW

**Comment by alikon** [comment]
i've applyed the patch
then i've runned npm ci
but uploading a large img give me this result:

Screenshot from 2020-09-05 09-49-26

what i'm missing ?

**Comment by maliouris** [comment]
> i've applyed the patch
> then i've runned npm ci
> but uploading a large img give me this result:
> 
> Screenshot from 2020-09-05 09-49-26
> 
> what i'm missing ?

Can you give me more info ? Because in my point of view I see this: 
Screenshot_2020-09-05 Media - Test - Administration

Did you run "npm run dev:com_media" and "npm run build:js" ? If you did just tell my all the steps to reproduce it.
(Also refresh your browser page CTL+F5 in order to load the compiled js)

**Comment by alikon** [comment]
ok after running
`npm run dev:com_media`
`npm run build:js`
plus
`CTL+F5`
i've got the expected results
 thank you..... maybe better to update the Testing Instructions
to help dumb people like me :smiley:

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4a699d4182970d6efa34170d3f897143c42e87e0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30573">issues.joomla.org/tracker/joomla-cms/30573</a>.</sub>

**Comment by Quy** [comment]
Message should not be in upper case.

**Comment by maliouris** [comment]
> Message should not be in upper case.

I resolved it. You can check again

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b4963eab3f5c02383e207de7442311d7bb326d2a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30573">issues.joomla.org/tracker/joomla-cms/30573</a>.</sub>

File: build/media_source/system/js/core.es6/text.es6 [hunk 1]
```diff
@@ -42,9 +42,9 @@
         Joomla.loadOptions({ 'joomla.jtext': null });
       }
 
-      newDef = newDef === undefined ? '' : newDef;
+      newDef = newDef === undefined ? newKey : newDef;
       newKey = newKey.toUpperCase();
-
+
```
Comment (Quy) [inline build/media_source/system/js/core.es6/text.es6]: Please remove spaces.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 627701558c8e479b9d6d54fe492e1a1ececd3ead<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30573">issues.joomla.org/tracker/joomla-cms/30573</a>.</sub>

**Comment by richard67** [comment]
@Quy 's test result is still valid, because changes after that was only code style (white space) and a clean update to the base branch. => 2 good tests => Ready to Commit.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30573">issues.joomla.org/tracker/joomla-cms/30573</a>.</sub>

**Comment by infograf768** [comment]
I just had to use npm ci

Thanks.

**Pull request merged**